File: api/src/main/java/cn/hirrao/javaee/entity/HeartRate.java
package cn.hirrao.javaee.entity;

import com.baomidou.mybatisplus.annotation.TableId;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
public class HeartRate {
    @TableId(value = "hrlId")
    private Long hrlId;
    private Long uid;
    private Float heartRate;
    private String recordTime;
}

File: api/src/main/kotlin/cn/hirrao/javaee/entity/HeartRate.kt
package cn.hirrao.javaee.entity

import com.baomidou.mybatisplus.annotation.TableId

data class HeartRate(
    @TableId(value = "hrlId") val hrlId: Long? = null,
    val uid: Long? = null,
    val heartRate: Float? = null,
    val recordTime: String? = null,
)